docs/pages/reference/mesheryctl-commands.md

```diff
@@ -1,401 +1,388 @@
-        <tr>
-          <td></td>
-          <td>{{ flag.name }}</td>
-          <td>{{ flag.description }}</td>
-      </tr>
-      {% endfor %}
-    {% endfor %}
-    {% for subcommand_hash in site.data.mesheryctlcommands.pattern.apply.commands %}{% assign subcommand = subcommand_hash[1] %}
-      <tr>
-        <td><a href="{{ site.baseurl }}/reference/mesheryctl/pattern/apply">{{ subcommand.name }}</a></td>
-        <td></td>
-        <td>{{ subcommand.description }}</td>
-      </tr>
-      {% for flag_hash in site.data.mesheryctlcommands.pattern.apply.flags %}{% assign flag = flag_hash[1] %}
-        <tr>
-          <td></td>
-          <td>{{ flag.name }}</td>
-          <td>{{ flag.description }}</td>
-      </tr>
-      {% endfor %}
-    {% endfor %}
-    {% for subcommand_hash in site.data.mesheryctlcommands.pattern.view.commands %}{% assign subcommand = subcommand_hash[1] %}
-      <tr>
-        <td><a href="{{ site.baseurl }}/reference/mesheryctl/pattern/view">{{ subcommand.name }}</a></td>
-        <td></td>
-        <td>{{ subcommand.description }}</td>
-      </tr>
-      {% for flag_hash in site.data.mesheryctlcommands.pattern.view.flags %}{% assign flag = flag_hash[1] %}
-        <tr>
-          <td></td>
-          <td>{{ flag.name }}</td>
-          <td>{{ flag.description }}</td>
-      </tr>
-      {% endfor %}
-    {% endfor %}
-    {% for subcommand_hash in site.data.mesheryctlcommands.pattern.delete.commands %}{% assign subcommand = subcommand_hash[1] %}
-      <tr>
-        <td><a href="{{ site.baseurl }}/reference/mesheryctl/pattern/delete">{{ subcommand.name }}</a></td>
-        <td></td>
-        <td>{{ subcommand.description }}</td>
-      </tr>
-      {% for flag_hash in site.data.mesheryctlcommands.pattern.delete.flags %}{% assign flag = flag_hash[1] %}
-        <tr>
-          <td></td>
-          <td>{{ flag.name }}</td>
-          <td>{{ flag.description }}</td>
-      </tr>
-      {% endfor %}
-    {% endfor %}
-  {% endfor %}
-</thead>
-</table>
+---
+layout: default
+title: Command Reference
+abstract: "A guide to Meshery's CLI: mesheryctl"
+permalink: reference/mesheryctl
+redirect_from: reference/mesheryctl/commands/
+type: Reference
+---
+## Categories and Command Structure
+
+Meshery CLI commands are categorized by function, which are:
+
+- `mesheryctl` - Global flags and CLI configuration
+- `mesheryctl system` - Meshery Lifecycle and Troubleshooting
+- `mesheryctl mesh` - Service Mesh Lifecycle & Configuration Management: provisioning and configuration best practices
+- `mesheryctl perf` -  Service Mesh Performance Management: Workload and service mesh performance characterization
+- `mesheryctl pattern` - Service Mesh Pattern Configuration & Management: Service mesh patterns and Open Application Model integration
+- `mesheryctl fitler` - Data Plane Intelligence: Registry and configuration of WebAssembly filters for Envoy

```

Comment:
Filter not fitler 😂

Reply:
I didn't do that :sweat_smile:

Reply:
@alphaX86 : I'm able to set that page manually, but does refactoring `YAML` works? If yes, then pl help me to figure out changes!

Reply:
Not sure though because even when we were setting up the pages for first time, that page alone got me a drift because of it's damn long command 😶

I mean to say that, even if we put examples, it'd be just a straight line and not as paragraph as we expect

Reply:
@alphaX86 : Can I make changes some changes, so you can better review? I think `manually` is the only way to add it!

Reply:
Go ahead

Reply:
If anything got worse better be ready to revert the change 🙂

Reply:
@alphaX86 : It's a bit gonna tricky, printed 4 times each commands xD, anyway to make it one..

Reply:
@alphaX86 : this issue get resolved.